core/iwasm/libraries/lib-wasi-threads/test/build.sh

```diff
@@ -18,18 +48,22 @@ for test_c in *.c; do
         thread_start_file=$WAMR_DIR/samples/wasi-threads/wasm-apps/wasi_thread_start.S
     fi
 
-    echo "Compiling $test_c to $test_wasm"
-    $CC \
-        -target wasm32-wasi-threads \
-        -pthread -ftls-model=local-exec \
-        -z stack-size=32768 \
-        -Wl,--export=__heap_base \
-        -Wl,--export=__data_end \
-        -Wl,--shared-memory,--max-memory=1966080 \
-        -Wl,--export=wasi_thread_start \
-        -Wl,--export=malloc \
-        -Wl,--export=free \
-        -I $WAMR_DIR/samples/wasi-threads/wasm-apps \
-        $thread_start_file \
-        $test_c -o $test_wasm
+    if [[   "$enable_stress_tests" = true   ]] || ! [[ " ${stress_tests[@]} " =~ " ${test_wasm} " ]]; then
+        echo "Compiling $test_c to $test_wasm"
+        $CC \
+            -target wasm32-wasi-threads \
+            -pthread -ftls-model=local-exec \
```

Comment:
is `-ftls-model=local-exec` needed here?

Reply:
it was not written by me and works like that for quite some time so I can't really say now. 

I will check with the author, but it shouldn't be related to this PR as this part is executed for all the existing tests